python/perf-kernels/streamk/README.md

```diff
@@ -0,0 +1,43 @@
+# streamk gemm script v0.1
```

Comment:
What would be needed to get it to 1.0?

Reply:
I need made it ready to explore half million benchmarks, and have a comparable performance with Tensile development

Reply:
I don't think we can have it comparable to tensile because that is outside of the scope of streamk. I think we can call this 0.1 until we have the wider tuning space working.